src/main/drivers/dma.h

```diff
@@ -22,6 +22,8 @@
 
 #include "resource.h"
 
+#ifdef USE_DMA // Should be gone?
```

Comment:
Shouldn't be needed - Betaflight coding standard is to not make headers conditional unless this is needed to make all cases build.